package.json

```diff
@@ -50,6 +50,7 @@
     "@github/clipboard-copy-element": "^1.1.2",
     "@github/details-menu-element": "^1.0.12",
     "@github/image-crop-element": "^5.0.0",
+    "@github/include-fragment-element": "^6.1.1",
```

Comment:
Is this used anywhere?

Reply:
@jonrohan I just added an import for it, but it's used to defer load of menu items.